### PR TITLE
Update 09-luis-garza.md

### PR DESCRIPTION
**Pull request by luis-garza**

### CODE REVIEW

File: _slides/09-luis-garza.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hey ho!
```
Comment (gusshawstewart) [inline _slides/09-luis-garza.md:6]: This a great change! 😄

File: _slides/09-luis-garza.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
```
Comment (gusshawstewart) [inline _slides/09-luis-garza.md:4]:
```suggestion
title: "Welcome to our slide deck!!!! This is an awesome deck!!!!" 
---
```

**Review by gusshawstewart — APPROVED** [review]
LGTM 🚢 🥇

File: _slides/09-luis-garza.md (hunk 2 above)
Comment (luis-garza) [reply _slides/09-luis-garza.md:4]: Hey, thanks for the fix!

File: _slides/09-luis-garza.md (hunk 1 above)
Comment (luis-garza) [reply _slides/09-luis-garza.md:6]: Of course... :smirk:

**Pull request merged**